src/js/view/MutationSummaryView.js

```diff
@@ -50,22 +50,59 @@ var MutationSummaryView = Backbone.View.extend({
 	render: function()
 	{
 		var self = this;
+        var cancerStudyId;
+        var patientSampleMap = {};
+        var patientIds = [];
+        if (PortalGlobals) {
+            cancerStudyId = PortalGlobals.getCancerStudyId();
+            patientSampleMap = PortalGlobals.getPatientSampleIdMap();
+            for (var i = 0; i < self.model.sampleArray.length; i++) {
+                patientIds.push(patientSampleMap[self.model.sampleArray[i]]);
+            }
+        }
+        else {
+            cancerStudyId = window.cancer_study_id;
+        }
+                
+        var args = {study_id:cancerStudyId, attribute_ids:["12_245_PARTC_CONSENTED"], patient_ids:patientIds};
+        var arg_strings = [];
+        for (var k in args) {
+            if (args.hasOwnProperty(k)) {
+                arg_strings.push(k + '=' + [].concat(args[k]).join(","));
+            }
+        }
+                
+        var arg_string = arg_strings.join("&") || "?";
+            
+        var mutationSummary;
+            
+        $.ajax({
+            type: "POST",
+            url: "api/clinicaldata/patients",
+            data: arg_string,
+            dataType: "json",
+            success: function (data) {
+                mutationSummary = self._germlineMutationSummary(data);
+            },
+            error: function (data) {
+                mutationSummary = self._mutationSummary();
+            },
+            complete: function () {
+                var variables = {
+                    mutationSummary: mutationSummary,
+                    geneSymbol: self.model.geneSymbol
+            };
```

Comment:
indentation after this point looks confusing but we can deal with it later while moving data retrieval/access function into a separate data proxy class